src/adapters/scene.js

```diff
@@ -87,7 +95,7 @@ Scene.prototype.initMapBox = async function (locationHash) {
     maxZoom: 20,
     interactive: window.no_ui ? false : true,
     locale,
-    ...(await this.getMapInitOptions(locationHash)),
+    ...(await this.getMapInitOptions({ locationHash, bbox })),
```

Comment:
`getMapInitOptions` is no longer async: `await` can be removed.

Reply:
Indeed, thank you.
To avoid another cycle of modify/review I'll merge as-is and remove the `await` in the related PR which touches this file.